data/mods/Magiclysm/effects/effects.json

```
@@ -196,6 +196,7 @@
     "apply_message": "You feel very seen!",
     "remove_message": "You no longer feel like somebody's watching you.",
     "rating": "good",
+    "max_duration": "5 m",
```

```suggestion
    "max_duration": "60 m",
```
Because the gas vents spawn randomly it is possible that someone could find gaps from inside the forge.   An hour should make that improbable.